src/index.ts

```diff
@@ -178,6 +178,14 @@ export const findNextItem = (stimuli: Stimulus[], theta = 0, method = 'MFI', dee
       nextStimulus: nextItem,
       remainingStimuli: arr,
     };
+  } else if (method === "random"){
+    const index = Math.floor(Math.random() * arr.length);
+    const nextItem = arr[index];
+    arr.splice(index, 1);
```

Comment:
```suggestion
    const nextItem = arr.splice(index, 1)[0];
```

I think these two lines could be combined.